interface/modules/custom_modules/oe-module-dorn/public/orders.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once "../../../../globals.php";
+
+ use OpenEMR\Common\Acl\AclMain;
+ use OpenEMR\Common\Csrf\CsrfUtils;
+ use OpenEMR\Common\Twig\TwigContainer;
+ use OpenEMR\Modules\Dorn\ConnectorApi;
+ use OpenEMR\Core\Header; //this is needed along with setupHeader() to get the pop up to appear
+
+$tab = "orders";
+$pageTitle = "DORN Orders";
+if (!AclMain::aclCheckCore('admin', 'users')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl($pageTitle)]);
+    exit;
+}
+$primaryInfos = ConnectorApi::getPrimaryInfos('');
+
+if (!empty($_POST)) {
+    if (isset($_POST['SubmitButton'])) { //check if form was submitted
+        $datas = ConnectorApi::searchOrderStatus($_POST['form_orderNumber'], $_POST['form_primaryId'], $_POST['form_startDateTime'], $_POST['form_endDateTime']);
+        if ($datas == null) {
+            $datas = [];
+        }
+    }
+}
+?>
+<html>
+<head>
+        <?php Header::setupHeader(); ?>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+<title> <?php echo xlt($pageTitle); ?>  </title>
+<script>
+
+</script>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                require '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <div class="card">
+                <div class="card-body">
+                    <h5 class="card-title"><?php echo xlt("DORN - Lab Orders"); ?></h5>
+                    <div class="row">
+                        <div class="col">
+                            <form method="post" action="orders.php">
+                                <div class="row">
+                                    <div class="col-md-4">
+                                        <select name="form_primaryId">
+                                            <?php foreach ($primaryInfos as $primaryInfo) {
+                                                $selected = $primaryInfo->primaryId === $_GET['form_primaryId'] ? "selected" : "";
+                                                ?>
+                                            <option value='<?php echo attr($primaryInfo->primaryId) ?>' <?php $selected ?>>
+                                                <?php echo xlt($primaryInfo->primaryName) ?>
```

Comment:
what is an example or 2 of what we are translating in above line?

Reply:
I don't think that should be translating.  I changed it to text()

Reply:
change above to text (not yet done)